packages/ibm-products/src/components/InlineTip/_inline-tip.scss

```diff
@@ -0,0 +1,228 @@
+//
+// Copyright IBM Corp. 2023, 2023
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+/**
+ * The InlineTip specifically is theme-agnostic, and so we have to 
+ * use color tokens to keep the colors static instead of theme tokens 
+ * that will change depending on the selected theme.
+ * 
+ * We are triggering a *large* amount of linting errors.
+ * So, we're using the following "disable" rules for the entire file 
+ * instead of many individual rules.
+ */
+
+/* stylelint-disable carbon/layout-token-use */
+/* stylelint-disable carbon/theme-token-use */
+/* stylelint-disable declaration-no-important */
+/* stylelint-disable function-no-unknown */
+/* stylelint-disable max-nesting-depth */
+
+// Standard imports.
+@use '../../global/styles/project-settings' as c4p-settings;
+@use '@carbon/react/scss/colors' as *;
+@use '@carbon/layout/scss/convert' as *;
+@use '@carbon/styles/scss/spacing' as *;
+@use '@carbon/styles/scss/theme' as *;
+@use '@carbon/styles/scss/type' as *;
+
+$block-class: #{c4p-settings.$pkg-prefix}--inline-tip;
+// Images and animation are the same fixed size
+$media-height: to-rem(118px);
+$media-width: calc(#{$media-height} * 2);
+$purple: #7f3ae7;
+
+.#{$block-class} {
+  position: relative;
+  display: flex;
+  background: linear-gradient(90deg, $blue-90 0%, $purple-70 100%);
+}
+// layout only: wide
+.#{$block-class}__wide {
+  padding: $spacing-05 $spacing-10 $spacing-05 $spacing-09;
+}
+// layout: wide, with image
+.#{$block-class}__wide.#{$block-class}__has-media {
+  padding-right: $spacing-09;
+}
+
+.#{$block-class}__wide.#{$block-class}__has-media .#{$block-class}__media {
+  padding-left: $spacing-10;
+}
+
+.#{$block-class}__wide.#{$block-class}__has-media .#{$block-class}__media img {
+  display: block;
+  height: $media-height;
+}
+
+.#{$block-class}__wide.#{$block-class}__has-media .#{$block-class}__media svg {
+  width: $media-width !important;
```

Comment:
Can you provide a higher specificity selector to avoid using `!important` here and elsewhere in this file?